emojis.json

```diff
@@ -45,12 +45,12 @@
     "category": "people"
   },
   "innocent": {
-    "keywords": ["face", "angel", "heaven", "halo"],
+    "keywords": ["face", "angel", "heaven", "halo", "innocent"],
```

Comment:
Oops sorry I didn't catch this before. `innocent` shouldn't be a keyword since this emoji is already called `innocent`.